draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -221,6 +228,10 @@ const IndicatorsContainer: typeof components.IndicatorsContainer = props => (
     className={styles.indicatorsContainer}
   />
 )
+
+const ValueContainer: typeof components.ValueContainer = props => (
+  <components.ValueContainer {...props} className={styles.valueContainer} />
+)
```

Comment:
Turns out targeting the who of the controls for the font update was a bad idea and was throwing off the layout on the reversed backgrounds. I've added this primitive in to supersede the original container class